src/cadenza.js

```diff
@@ -133,6 +133,18 @@ globalThis.cadenza = Object.assign(
  * _Note:_ Since numbers in JavaScript are Double values ([more info on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number#number_encoding)),
  * for Long variables, the API is currently limited to the Double value range.
  */
+/**
+ * @typedef Feature - A [GeoJSON](https://geojson.org/) feature object
+ * @property {any[]} id - The id of the feature
```

Comment:
This definition is not according to the [spec](https://datatracker.ietf.org/doc/html/rfc7946#section-3.2):
> If a Feature has a commonly used identifier, that identifier SHOULD be included as a member of the Feature object with the name "id", and the value of this member is either a JSON string or number.

Maybe we should model this as a [foreign member](https://datatracker.ietf.org/doc/html/rfc7946#page-15) instead, e.g. "cadenzaObjectId"?

Reply:
I can add it as a foreign member. Then i would not provide a actually ID, since i do not have one.
Right?

Reply:
Yes, I think that would be better to not conflict with the spec :+1:

Reply:
Andreas have a opinion about the id. Perhaps we discuss this again on Monday

Reply:
we choose the name objectId